deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py

```diff
@@ -158,7 +125,7 @@ class PropertySegmentsPerformance(WeakSegmentsAbstractText):
         Properties to check, if none are given checks all properties except ignored ones.
     ignore_properties : Union[Hashable, List[Hashable]] , default: None
         Properties to ignore, if none given checks based on properties variable
-    n_top_properties : int , default: 10
+    n_top_properties : Optional[int] , default: 10
```

Comment:
I don't think that you need to write this in the docstring, default: 10 means it's Optional already